Makefile

```diff
@@ -20,12 +20,12 @@ endif
 	poetry config virtualenvs.in-project true
 	poetry dynamic-versioning show || poetry sync --only-root
 	poetry sync
-	poetry run playwright install firefox
+	poetry run python -m playwright install firefox
 
 	test -f $(OTTERDOG_LINK) || ln -s $(OTTERDOG_SCRIPT) $(OTTERDOG_LINK)
 
 test:  ## Run tests
-	poetry run py.test
+	poetry run python -m pytest
```

Comment:
does `poetry run pytest` work on Mac? I am all for replacing the deprecated `py.test`, but when the package is installed in the virtual environment, there should be a script called `pytest`.

Reply:
Yes, `poetry run pytest` works on Mac. I just tested it.

Reply:
Sorry I misread the comments yesterday. So to be clear.

* `poetry run pytest` did not work on my system. The pytest binary appears to be missing.
* `poetry run python -m pytest` did work for me which is why I proposed this change.

However I'm not using python installed from Homebrew. I use pyenv to install Python I'm not sure how that affects how poetry finds its binaries inside it' own virtual environment.